src/dsa/rpo_falcon512/polynomial.rs

```diff
@@ -1,5 +1,6 @@
-use super::{FalconError, Felt, Vec, LOG_N, MODULUS, MODULUS_MINUS_1_OVER_TWO, N, PK_LEN};
+use super::{FalconError, Felt, LOG_N, MODULUS, MODULUS_MINUS_1_OVER_TWO, N, PK_LEN};
 use core::ops::{Add, Mul, Sub};
+use winter_utils::collections::*;
```

Comment:
Instead of importing from `winter_utils`, should we import from `crate::utils`? This way, we'll have more flexibility in how re-exports work in the future.